src/Nethermind/Nethermind.Network.Discovery/NettyDiscoveryHandler.cs

```diff
@@ -183,16 +182,43 @@ private byte[] Serialize(DiscoveryMsg msg)
     {
```

Comment:
Add a comment in a method description why we have that method although there are no references to it and we use the other one instead.

Reply:
I will probably remove this function instead - it was used earlier and now its not used.